lib/sqlalchemy/dialects/postgresql/types.py

```diff
@@ -247,6 +247,14 @@ def as_generic(self, allow_nulltype: bool = False) -> sqltypes.Interval:
     def python_type(self) -> Type[dt.timedelta]:
         return dt.timedelta
 
+    def literal_processor(self, dialect):
+        def process(value):
+            if value is not None:
+                value = f"'{str(value).replace(' days,', '')}'::interval"
```

Comment:
seems we can also just do `f"'{value.total_seconds()}'::interval`

Reply:
`f"make_interval(secs=>{value.total_seconds()})"` works too

Reply:
Updated as per comments, went with `make_interval` method.
i was trying to write tests for execution of queries but was getting this error, have added for compile as of now, not able to figure out how to make tests point to specific db hosted in my local for testing pupose
`test.dialect.oracle.test_types.TypesTest' unsupported on any DB implementation 'sqlite(3.31.1)+pysqlite'`

Reply:
by default the query runs using sqlite, that can't execute oracle specific code, so skip are automatically added when tests are marked as "backend".

You can use oracle via docker with the tests, check the readme unit test: https://github.com/sqlalchemy/sqlalchemy/blob/main/README.unittests.rst#docker-configurations
or I can just make it run the pipeline on gerrit ;)

Reply:
mypy seems the only failure here. It logged
```
pep484: commands[0]> mypy ./lib/sqlalchemy
lib/sqlalchemy/dialects/postgresql/types.py:250: error: Function is missing a type annotation  [no-untyped-def]
lib/sqlalchemy/dialects/postgresql/types.py:251: error: Function is missing a type annotation  [no-untyped-def]
Found 2 errors in 1 file (checked 252 source files)
```

Reply:
Yeah i have added typing to method(s) but for some reason couldn't figure out how to resolve this, without modifying superclass returntype for now added no_type_check decorator ( saw that oracle had it for entire file, so added this for postgres in method only, and to keep it consistent also added it in oracle )
```py
error: Return type "Callable[[Optional[timedelta]], Optional[str]]" of "literal_processor" incompatible with return type "Optional[_LiteralProcessorType[timedelta]]" in supertype "TypeEngine"  [override]
```
added testcase for db specific backend testing also.